### PR TITLE
cyclic sort using c++ lang

### PR DESCRIPTION
**Pull request by vijay-kumar-yadav**

### CODE REVIEW

**Review by Kumar-laxmi — COMMENTED** [review]
Please update the **README.md**

**Comment by vijay-kumar-yadav** [comment]
> Please update the **README.md**
Done ✔️

File: README.md [hunk 1]
```diff
@@ -48,8 +48,9 @@
...
 | 13.  | Selection Sort | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Sorting/selection_sort.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/selection_sort.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Sorting/selection_sort.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Sorting/selection_sort.py) | Sorting |
-| 14.  | AVL Tree | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/AVLTree.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/AVLTree.py) | Trees |
-| 15.  | Decision Tree | [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Machine%20Learning/Decision%20Tree/decision_tree.py) | Machine Learning |
+ | 14.  | Cycle Sort |  [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/cycle_sort.cpp) | Sorting |
```
Comment (Kumar-laxmi) [inline README.md:51]: Place **Cyclic Sort** between **Bubble Sort** and **Heap Sort**

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vijay-kumar-yadav) [reply README.md:51]: ok

Comment (vijay-kumar-yadav) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -44,12 +44,14 @@
...
-| 13.  | Selection Sort | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Sorting/selection_sort.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/selection_sort.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Sorting/selection_sort.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Sorting/selection_sort.py) | Sorting |
-| 14.  | AVL Tree | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/AVLTree.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/AVLTree.py) | Trees |
-| 15.  | Decision Tree | [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Machine%20Learning/Decision%20Tree/decision_tree.py) | Machine Learning |
+| 10.  | Cycle Sort |  [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/cycle_sort.cpp) | Sorting |
```
Comment (Kumar-laxmi) [inline README.md:47]: Screenshot 2023-01-02 at 8 16 10 AM

It is leading to some error in table, please rectify it

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vijay-kumar-yadav) [reply README.md:47]: fixed

**Comment by Kumar-laxmi** [comment]
@vijay-kumar-yadav Kindly resolve branch conflict

**Pull request closed without merging**